src/doc/src/reference/profiles.md

```diff
@@ -117,7 +117,9 @@ strip = "debuginfo"
 ```
 
 Possible string values of `strip` are `"none"`, `"debuginfo"`, and `"symbols"`.
-The default is `"none"`.
+The default is `"none"` by default, and `"debuginfo"` in release profile.
+Be aware that stripping `"symbols"` may make backtraces incomprehensible,
+depending on the platform.
```

Comment:
For the most part, the cargo documentation doesn't dive into the particulars of explaining the different options and their consequences. That documentation should generally live in the rustc book (so we don't have to duplicate it in two remote places). Adding warnings about the consequences of stripping symbols would probably be appropriate over there (and seems good to include).

Reply:
It is my experience that people often do not actually click through, simply put.